### ISSUE TITLE
Custom maps don't load when joining a server if you have the mod unticked.

### ISSUE THREAD

**Issue by DanielWillett** [SmartlyDressedGames/Unturned-3.x-Community#4087]
Video:
https://youtu.be/dsN_UJIP7qQ

(At the end of this video I still get kicked but it's for an unrelated workshop issue I didn't feel like fixing, it at least let me past the map stage.)

When you join a server, mods you have unchecked in the 'Workshop > Manage' tab will load, but if the server is running a custom map and the mod for that map isn't checked, an error will throw saying you don't have the map installed (even if it is cached by Steam).

I think the intended behavior is that all the mods should load before checking if the map is installed or not.

**Comment by SDGNelson** [comment]
You're right that this should probably bypass the workshop file on/off setting. I'll take care of this as part of: #3990

**Comment by DanielWillett** [comment]
ty

**Comment by DanielWillett** [comment]
By the way not sure if you meant to fix this with #3990 but if so it still happens.

**Comment by SDGNelson** [comment]
Sorry, I guess I didn't end up changing that behavior as part of #3990 - it tried to preserve the existing behavior as much as possible. All of those methods were consolidated to pull from a list of scanned maps, and during scanning it checks whether the workshop file is enabled. Maybe all levels should be added to the list but only enabled levels shown in the UI? 🤔

**Comment by GazziFX** [comment]
I think assets of disabled maps shouldn't be loaded until you join this map
